pkg/collector/dist/datadog.yaml

```diff
@@ -61,6 +61,9 @@ api_key:
 # WARNING: disabling it on every agent will lead to display and billing issues
 # enable_metadata_collection: true
 
+# Enable the gohai collection of systems data
+# enable_gohai: false
```

Comment:
Default value is `true`, we should show it in the default conf

Reply:
I fixed it! Thank you!